helmchart/values.yaml

```diff
@@ -0,0 +1,71 @@
+# Default values for httpbin.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: gcr.io/mvno-e3a61b6e-3/httpbin
```

Comment:
This should be a pointer to Docker Hub, I think.

Reply:
Ah yes. Since the chart has readonly fs and no root, I couldn't point to the official image. So had it pointing to my own non public repo. Changed it to a public docker hub build.